framework/test/jest/unit/specs/controller/bus.spec.js

```diff
@@ -85,7 +85,18 @@ describe('Bus', () => {
 			const actions = ['action1', 'action2'];
 
 			// Act
-			await bus.registerChannel(moduleAlias, [], actions);
+			await bus.registerChannel(moduleAlias, [], actions, {
```

Comment:
For the sake of readability, please move this object to a top level variable, set it in a beforeEach hook and reuse that variable, mutate it in an `Arrange` section if needed.